src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  margin-top: 16px;
+  align-items: flex-end;
+  text-align: justify;
```

Comment:
The property 'text-align: justify;' is not applicable to flex containers. Consider removing it or applying it to a child element if needed.